app/Models/Store/Order.php

```diff
@@ -184,6 +185,23 @@ public function scopeWithPayments($query)
         return $query->with('payments');
     }
 
+    #region Shopify convenience setters for webhook updates
+    public function setAdminGraphqlApiIdAttribute(string $value)
+    {
+        $this->reference = $value;
+    }
+
+    public function setOrderNumberAttribute(string $value)
+    {
+        $this->transaction_id = static::PROVIDER_SHOPIFY."-{$value}";
```

Comment:
`.'-'.$value` isn't all that much different and more "uniform" in term of the string concat method 🙃